app/views/overview/_service-instance-row.html

```diff
@@ -0,0 +1,133 @@
+      <div style="width: 300px">
+        <div>
+          <strong ng-if="!row.instanceBindings.length">No Bindings</strong>
+          <strong ng-if="row.instanceBindings.length">Bindings</strong>
+        </div>
+        <a g-if="row.instanceBindings.length >= 1" href="{{row.getSecretForBinding(row.instanceBindings[0]) | navigateResourceURL}}">{{row.instanceBindings[0].metadata.name}}</a><span ng-if="row.instanceBindings.length > 1">,&nbsp;</span>
+        <a
+          ng-if="row.instanceBindings.length > 1"
+          ng-click="row.toggleExpand($event, true)">
+          {{row.instanceBindings.length -1}} others</a>
+      </div>
+    </div>
+
+    <div
+      class="hidden-xs hidden-sm"
+      ng-if="row.apiObject.spec.osbDashboardURL">
+      <span class="page-header-link">
+        <a ng-href="{{row.apiObject.spec.osbDashboardURL}}" target="_blank">
+          Console <i class="fa fa-external-link" aria-hidden="true"></i>
+        </a>
+      </span>
+    </div>
+
+    <div class="list-pf-actions">
+      <div uib-dropdown>
+        <a href=""
+        uib-dropdown-toggle
+        class="actions-dropdown-kebab"><i class="fa fa-ellipsis-v"></i><span class="sr-only">Actions</span></a>
+        <ul class="dropdown-menu dropdown-menu-right" uib-dropdown-menu role="menu">
+          <!-- TODO: follow-on PR
+          <li role="menuitem">
+            <a href="" ng-click="row.bind()" role="button">Bind to Application</a>
+          </li>
+          <li role="menuitem">
+            <a href="" ng-click="row.unBind()" role="button">Un-bind</a>
+          </li>
+          -->
+          <li role="menuitem">
+            <a href="" ng-click="row.deprovision()" role="button">Deprovision</a>
+          </li>
+        </ul>
+      </div>
+    </div>
+
+  </div>
+
+  <div
+    class="list-pf-expansion collapse"
+    ng-if="row.expanded"
+    ng-class="{ in: row.expanded }">
+
+    <div class="list-pf-container">
+
+      <!--
+        TODO: configuration details may not be possible for MVP
+      <div class="expanded-section">
+        <div class="section-title">
+          Configuration Details
+        </div>
+        <div class="row" ng-repeat="item in [{
+          name: 'password',
+          value: '*****'
+        }, {
+          name: 'lorem ipsum',
+          value: 'dolor sit amet'
+        }]">
+          <div class="col-sm-5">{{item.name}}</div>
+          <div class="col-sm-7">{{item.value}}</div>
+        </div>
+      </div>
+      -->
+
+      <div class="expanded-section">
+        <div class="section-title">
+          Bindings
+        </div>
+        <span ng-if="!row.instanceBindings.length">There are no bindings.</span>
+        <div
+          ng-if="row.instanceBindings.length"
+          class="row"
+          ng-repeat="binding in row.instanceBindings">
+          <div class="col-sm-4">
+
+            <a href="{{row.getSecretForBinding(binding) | navigateResourceURL}}">
+              {{binding.metadata.name}}
```

Comment:
Yeah that seems misleading/cause confusion.  So list the binding, no link, second 'view secret' link seems much better. I'll update.